fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -270,6 +270,10 @@ private void setUnUsedOutputColumns(PhysicalOlapScanOperator node, OlapScanNode
                                             List<ScalarOperator> predicates, OlapTable referenceTable) {
             if (ConnectContext.get().getSessionVariable().isAbleFilterUnusedColumnsInScanStage()) {
```

Comment:
```
if (!ConnectContext.get().getSessionVariable().isAbleFilterUnusedColumnsInScanStage()) {
    return;
}
```
I think this will make code easier to read.